macros/src/derive_fn.rs

```diff
@@ -164,6 +177,21 @@ pub fn derive_fn(
             ) -> stellar_contract_sdk::RawVal {
                 call_raw(env, #(#slice_args),*)
             }
+
+            #[cfg(feature = "testutils")]
+            #[cfg_attr(feature = "docs", doc(cfg(feature = "testutils")))]
+            pub fn call_external(
+                e: &mut stellar_contract_sdk::Env,
+                contract_id: &stellar_contract_sdk::Binary,
+                #(#invoke_args),*
+            ) {
+                e.invoke_contract(
+                    stellar_contract_sdk::xdr::HostFunction::Call,
+                    (contract_id, #wrap_export_name, #(#invoke_idents),*).try_into().unwrap()
+                )
+                .try_into()
+                .unwrap()
```

Comment:
We might want to return the `Result` given that the function will be used in testing, there's a good chance people will be asserting on the return value, and it is easy to assert on a `Result` which makes testing a bit clearer imo.

Reply:
Opened #330